macros/showMore.njk

```diff
@@ -0,0 +1,13 @@
+{% macro showMore(label, content) %}
+  <div class="show-more">
+    <div class="bold-small">{{ t(label) }}</div>
+    {% if longText.length > 400 %}
+      <input id="more-less" type="checkbox">
+    {% endif %}
+    <p class="form-group-compound">{{ content }}</p>
+    {% if longText.length > 400 %}
+      <label class="link more" for="more-less">{{ t('Show more') }}</label>
+      <label class="link less" for="more-less">{{ t('Show less') }}</label>
```

Comment:
Shouldn't one of these links be hidden by default?

Reply:
this uses css to hide and show and does not rely on javascript. The css to control it is in the citizen frontend